feedback-module/src/components/Module.js

```diff
@@ -122,9 +119,18 @@ function Module({ pagetitle, endpoint, dir }) {
       pageTitleAsScreen(currentPageTitle);
       pageChange(currentPageTitle, nextPageTitle);
 
-      sendFormData(screen.formID),
-        setScreen(SCREENS[nextScreen]),
-        setCheckboxError(false);
+      if (requestInfo(screen.formID)) {
+        setLoading(true);
+        requestService(
+          ...requestInfo(screen.formID),
+          () => {
+            setScreen(SCREENS[nextScreen]);
+            headerRef.current.scrollIntoView(true);
+          },
+          setFailedRequest,
```

Comment:
Don't these need arguments?

Reply:
I'm passing them as arguments to `requestService`, and I call them with arguments there.